src/controller/audio-stream-controller.js

```diff
@@ -446,6 +444,7 @@ class AudioStreamController extends BaseStreamController {
       newDetails.PTSKnown = false;
     }
     track.details = newDetails;
+    this.trackLastLoaded = trackId;
```

Comment:
@realeyes-matthew This is were the track gets set with new details when an audio track is loaded.

Reply:
Great, merging this into the rest of the CMAF PR